src/main/kotlin/app/revanced/patches/youtube/ad/general/bytecode/patch/GeneralBytecodeAdsPatch.kt

```diff
@@ -357,7 +357,9 @@ class GeneralBytecodeAdsPatch : BytecodePatch() {
                                     lithoMethod.addInstructions(
                                         insertIndex, // right after setting the component.pathBuilder field,
                                         """
-                                                invoke-static {v5, v2}, Lapp/revanced/integrations/patches/LithoFilterPatch;->filter(Ljava/lang/StringBuilder;Ljava/lang/String;)Z
+                                                move-object/from16 v$clobberedRegister, v28
```

Comment:
Why not use a p alias here?

Reply:
> Why not use a p alias here?

Because invoke-static/range {p3 ... p3} instruction is mapped on v28 (based on the patcher).

Reply:
If p3 is mapped to v28, and you use v28, you can also use p3, it is less arbitrary and automatically mapped to the correct register.

Reply:
> If p3 is mapped to v28, and you use v28, you can also use p3, it is less arbitrary and automatically mapped to the correct register.

When i tried to do it, it didn't work.